android/src/main/java/com/revenuecat/purchases/react/RNPurchasesModule.java

```diff
@@ -82,23 +81,14 @@ public void setupPurchases(String apiKey, @Nullable String appUserID,
                                boolean observerMode, @Nullable String userDefaultsSuiteName) {
         PlatformInfo platformInfo = new PlatformInfo(PLATFORM_NAME, PLUGIN_VERSION);
         CommonKt.configure(reactContext, apiKey, appUserID, observerMode, platformInfo);
-        Purchases.getSharedInstance().setUpdatedPurchaserInfoListener(this);
+        Purchases.getSharedInstance().setUpdatedCustomerInfoListener(this);
     }
 
     @ReactMethod
     public void setAllowSharingStoreAccount(boolean allowSharingStoreAccount) {
         CommonKt.setAllowSharingAppStoreAccount(allowSharingStoreAccount);
     }
 
-    @ReactMethod
-    public void addAttributionData(ReadableMap data, Integer network, @Nullable String networkUserId) {
```

Comment:
Looks like this is gone too?

Reply:
yes

Reply:
I'll remove it from iOS too then.

Reply:
https://github.com/RevenueCat/purchases-hybrid-common/pull/169